packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

```diff
@@ -159,6 +149,161 @@ You can also delete these assets using the following requests (**Delete logo**,
+
+<ApiOperation method="get" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations" />
+
+This operation returns a [paginated](/docs/reference/core-okta-api/#pagination) list of [email customization](/docs/reference/api/brands/#email-customization) resources.
+
+#### Create email customizations
+
+You can create a new email customization (**Create email customization** in Postman).
+
+<ApiOperation method="post" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations" />
+
+The request body needs an [email customization](#email-customization) resource. The operation returns the created email customization.
+
+> **Note:** If this is the first customization being created for the email template, `isDefault` is set to `true`.
+
+#### Delete the customizations of an email template
+
+You can delete all customizations made to a specific email template (**Delete all email customizations** in Postman).
+
+<ApiOperation method="delete" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations" />
+
+On success, this operation returns a `204 No Content` message.
+
+#### Get an email customization
+
+You can fetch a specific email customization (**Get email customization** in Postman).
+
+<ApiOperation method="get" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations/${customizationId}" />
+
+This operation returns the requested [email customization](/docs/reference/api/brands/#email-customization) resource.
+
+#### Update an email customization
+
+You can update a specific email customization (**Update email customization** in Postman).
+
+<ApiOperation method="put" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations/${customizationId}" />
+
+The request body needs an [email customization](#email-customization) resource. The operation returns the updated email customization.
+
+> **Note:** If `isDefault` is `true`, the previous default email customization will have its `isDefault` set to `false`.
+
+#### Delete an email customization
+
+You can delete a specific customization made to an email template (**Delete email customization** in Postman).
+
+<ApiOperation method="delete" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations/${customizationId}" />
+
+On success, this operation returns a `204 No Content` message.
+
+If the email customization to be deleted is the default, this operation returns a `409 Conflict` message.
+
+#### Preview an email customization
+
+You can fetch a preview of the customizations of an email template (**Preview email customization** in Postman).
+
+<ApiOperation method="get" url="/api/v1/brands/${brandId}/templates/email/${templateName}/customizations/${customizationId}/preview" />
+
+This operation returns the [default content](/docs/reference/api/brands/#email-content) resource of the specified email template, with the variables populated with the current user's context.
```

Comment:
Should we capitalize the names of the resources - e.g., `Default Content` resource, `Email Customization` resource?

Reply:
that is currently not a part of our style guidelines. we only capitalize the names of objects. we try to avoid over capitalization, if possible.